src/strings/csv/en.csv

```diff
@@ -732,15 +733,25 @@ REPORT_COULD_NOT_SAVE,Could not save report,
 REPORT_COULD_NOT_SUBMIT,Could not submit report,
 REPORT_COULD_NOT_UNLOCK,Could not unlock report,
 REPORT_EDIT,Edit Report,
+REPORT_MORTALITY_RATE_INFO,If known please list the death rates of all planted trees and other plants as a percentage. If unknown please leave blank,
 REPORT_NAME,Report Name,
 REPORT_NO_NURSERIES,No nurseries included in report.,
 REPORT_NO_PLANTING_SITES,No planting sites included in report.,
 REPORT_NO_SEEDBANKS,No seed banks included in report.,
+REPORT_NURSERY_CAPACITY_INFO,Please list the total number of plants that the nursery can hold using best estimates.,
 REPORT_PROBLEM,Report Problem,
 REPORT_SAVE_AND_CLOSE,Save and Close,
 REPORT_SEEDBANK_BUILD_COMPLETION_DATE,Seed Bank Build Completion Date *,
 REPORT_SEEDBANK_BUILD_START_DATE,Seed Bank Build Start Date *,
 REPORT_SEEDBANK_OPERATION_START_DATE,Seed Bank Operation Start Date *,
+REPORT_SUMMARY_OF_PROGRESS_INFO,Give a brief summary of progress including advancement toward meeting restoration targets as well as social and community benefits in the project.,
+REPORT_TOTAL_NON_TREES_PLANTED_INFO,Please include all other plants planted that are affiliated with the Terraformation project; do not include trees in this figure.,
+REPORT_TOTAL_PAID_WORKERS_INFO,Please list the total number of full-time or part-time paid workers engaged in planting in this project,
```

Comment:
end with period (and on line 751) or remove them from others?

Reply:
Good find. I'll go ahead and change that. Thanks